SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -361,113 +366,61 @@ void FLocalDeploymentManager::SetAutoDeploy(bool bInAutoDeploy)
 }
 
 // TODO: UNR-4423 Change this to taking a snapshot without the services
-void SPATIALGDKSERVICES_API FLocalDeploymentManager::TakeSnapshot(UWorld* World, bool bUseStandardRuntime,
-																  FSpatialSnapshotTakenFunc OnSnapshotTaken)
+void SPATIALGDKSERVICES_API FLocalDeploymentManager::TakeSnapshot(UWorld* World, FSpatialSnapshotTakenFunc OnSnapshotTaken)
 {
 	FHttpModule& HttpModule = FModuleManager::LoadModuleChecked<FHttpModule>("HTTP");
 	TSharedRef<IHttpRequest> HttpRequest = HttpModule.Get().CreateRequest();
 
-	HttpRequest->OnProcessRequestComplete().BindLambda(
-		[World, bUseStandardRuntime, OnSnapshotTaken](FHttpRequestPtr HttpRequest, FHttpResponsePtr HttpResponse, bool bSucceeded) {
-			if (!bSucceeded)
+	HttpRequest->OnProcessRequestComplete().BindLambda([World, OnSnapshotTaken](FHttpRequestPtr HttpRequest, FHttpResponsePtr HttpResponse,
+																				bool bSucceeded) {
+		if (!bSucceeded)
+		{
+			UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Failed to trigger snapshot at '%s'; received '%s'"), *HttpRequest->GetURL(),
+				   *HttpResponse->GetContentAsString());
+			if (OnSnapshotTaken != nullptr)
 			{
-				UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Failed to trigger snapshot at '%s'; received '%s'"),
-					   *HttpRequest->GetURL(), *HttpResponse->GetContentAsString());
+				OnSnapshotTaken(false /* bSuccess */, FString() /* PathToSnapshot */);
+			}
+			return;
+		}
+
+		// Unfortunately by the time this callback happens, the files haven't been flushed, so if you copy you may get
```

Comment:
This should no longer be needed, right?

Reply:
I'll try without, fair point. Not sure on the semantics of the new snapshotting

Reply:
Confirmed with runtime, you're correct. Removing the timer.